src/config/fonts.js

```diff
@@ -6,3 +6,12 @@ export const fonts = {
   fontPrimaryBold: { fontFamily: 'Nunito-Bold' },
   fontCode: { fontFamily: 'Menlo' },
 };
+
+export const fontsStyled = {
```

Comment:
I think @josenaranjo is working on something related to fonts as he mentioned [here](https://github.com/gitpoint/git-point/issues/503#issuecomment-337936548) AFAIK he created a new file called `styledFonts.js` inside `/config`, maybe you can talk to him before merging this PR so you don't get any merge conflict in the future. FWIW I like the idea of having a new fresh file with fonts.

Reply:
`styledFonts.js` looks good to me. I'll wait on @josenaranjo PR to get merged before changing this one

Reply:
@machour ok, @josenaranjo's PR was merged https://github.com/gitpoint/git-point/pull/561 You can start using his font file.

Reply:
🤔  I actually like having all the fonts in one file. Not sure though.

Reply:
seconding @andrewda, keeping both variables in the same file will help making sure we change both when updating something.

@alejandronanez  Is it okay of I merge it back in `fonts.js` ?

Reply:
Sounds good to me, do you mind removing the file @josenaranjo added on his PR and doing a small refactor on his component so we don't introduce more tech debt in here?